www-src/app/locales/de.json

```diff
@@ -14,21 +15,24 @@
     "cozy notifications sync label": "Sync Cozy Benachrichtigungen ",
     "home": "Start",
     "about": "Über",
-    "last backup": "Zuletzt war:",
+    "last backup": "Last was:",
     "reset title": "Reset",
     "reset action": "Reset",
     "retry synchro": "Sync",
-    "synchro warning": "Es wird alles neu synchronisiert. Es kann längere zeit dauern. ",
-    "reset warning": "Es wird alle Cozy Daten auf deinem Handy löschen",
```

Comment:
Nevermind, this is normal transifex behavior caused by changed english sentences (as explained on IRC)